flink-clients/src/main/java/org/apache/flink/client/cli/CommandLineOptions.java

```diff
@@ -26,15 +26,19 @@
  * Base class for all options parsed from the command line.
  * Contains options for printing help and the JobManager address.
  */
-public abstract class CommandLineOptions {
+public class CommandLineOptions {
```

Comment:
Why we remove `abstract`?

Reply:
Thanks for your review! 

This PR adds a new interface `CommandAction`, and its `CommandAction#parse` method parses command line args to `CommandLineOptions`, but for `--version` or `--help` command, there is no suited type of `CommandLineOptions`, like `StopOptions` for `stop` command, so I removed `abstract` to create `CommandLineOptions` instances directly for `--help` and `--version` commands.